docs/setup_installation/aws/getting_started.md

```diff
@@ -4,6 +4,11 @@
 in the cloud. It integrates seamlessly with third-party platforms such as Databricks,
 SageMaker and KubeFlow. This guide shows how to set up [managed.hopsworks.ai](https://managed.hopsworks.ai) with your organization's AWS account.
 
+## Prerequisit
+To run the commands in this guide, you must have the AWS CLI installed and configured. See the [Getting started guide](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-install.html) in the AWS CLI User Guide for more information.
```

Comment:
we should add that these commands are using the default profile and they should use `--profile` if another profile is used

Reply:
It's added to the other pull request. This will be fixed when I rebase one on the other.